### PR TITLE
Fix UtxoInterface and test via fetchPrevoutAndTryToUnblindUtxo method

### PR DESCRIPTION
**Pull request by Janaka-Steph**
The return value from Esplora has changed but UtxoInterface has not been updated. 
This PR add the missing fields and test the interface via fetchPrevoutAndTryToUnblindUtxo method.

Please review @tiero @louisinger

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -34,6 +34,15 @@ export interface UtxoInterface {
   txid: string;
   vout: number;
   asset?: string;
+  assetcommitment?: string;
```
Comment (tiero) [inline src/types.ts:37]: I'm not sure this type is the same type you are thinking of. AFAIK this is only used internally, it should not be used to  Esplora JSON response

**Comment by tiero** [comment]
This opens up a good point: is the current design of `UtxoInterface` covering to much?

ie. it has fields which are used internally (ie. unblindData, prevout) menwhile is also used as return type of `fetchPrevoutAndTryToUnblindUtxo`

File: src/types.ts (hunk 1 above)
Comment (tiero) [reply src/types.ts:37]: nvm, this is actually ok.

**Review by tiero — CHANGES_REQUESTED** [review]
Discard all previous comments:

1. ACK for EsploraTxInterface  defined in src/esplora/types. Please add it as return type for the method [fetchUtxos](https://github.com/vulpemventures/ldk/blob/6e70276fb169f45f16cc976281c02fe62c2d893f/src/explorer/esplora.ts#L47)
2. nACK for adding status and commitment into UtxoInterface
3. ACK for adding the test  should have fetchPrevoutAndTryToUnblindUtxo return correct utxo interface  in test/esplora.test.ts , which should be renamed to `should return an UtxoInterface with extra esplora enriched fields if the UtxoInterface interface as input contains extra esplora enriched fields`

**Review by tiero — APPROVED** [review]

**Review by louisinger — APPROVED** [review]

**Pull request merged**